influxdb3_processing_engine/src/plugins.rs

```diff
@@ -718,7 +718,7 @@ mod python_plugin {
     }
 
     enum Schedule {
-        Cron(OwnedScheduleIterator<Utc>),
+        Cron(Box<OwnedScheduleIterator<Utc>>),
```

Comment:
clippy got smart about significant difference in size of variants and suggested boxing instead.

It said that `Cron(OwnedScheduleIterator<Utc>)` required 288 bytes, whereas `Every(Duration)` was 16 bytes. This reduces the enum to 16 bytes.